### DISCUSSION TITLE
OCI: Permission Denied, again (And solution)

### DISCUSSION THREAD

**Discussion by Bubbelb** [89luca89/distrobox#1187] [General]
**Description**
Error when entering a (alpine) container, with an "OCI: Permission Denied"

Issue found and worked around (read further).

**To Reproduce**

On an Archlinux (Manjaro) computer:
- Create normal user (userx)
- Set user with uncommon UID/GIC combo (1000:1001)
- Setup subuid/subgid as normal (userx:327680:65535 for both)
- distrobox create -i alpine -n alpine
- distrobox enter alpine
- ERROR

**Expected behavior**
A normal container start.

**Workaround**
- log out of userx
- log in as root
- remove /home/userx/.local/share/containers (don't know if this is 100% needed)
- usermod -u 1001 userx
- chown -R userx:userx (Might not be needed, since usermod should do this)
- re-login as userx (I rebooted to be sure)
- retry to create and enter alpine container... works

**Desktop (please complete the following information):**
 - Are you using podman, docker or lilipod? podman
 - Which version or podman, docker or lilipod? 4.8.2
 - Which version of distrobox? 1.6.0.1
 - Which host distribution? Manjaro-Gnome (Latest)
 - How did you install distrobox? package manager

**Additional context**
This uid/gid being off was a result of a test I did a while ago and never thought about it. It is definitely not a normal configuration.
I assume the problem lies in an assumption in the code that uid=gid, always. I'm not a programmer, so I cannot pinpoint where the problem exactly is (even if it's in Distrobox, or in Podman for that matter).
Maybe the easiest thing to do is warn the user of this problem and how to fix it, using the workaround...

Hope this all helps :)

**Comment by 89luca89** [comment]
Thanks for the report @Bubbelb :smile: 

Maybe I'll add this to some docs, just in case, I'll convert this to a discussion so that in future people will be able to find it
